app/controllers/comments_controller.rb

```diff
@@ -2,17 +2,27 @@ class CommentsController < ApplicationController
   load_and_authorize_resource
   before_action :find_post
 
+  def index
+    @comments = @post.comments
+    respond_to do |format|
+      format.html
+      format.json { render json: @comments }
+    end
```

Comment:
- [ ] The API endpoint to list all the posts' comments isn't functioning as expected due to Devise authentication(see image). Kindly check and make sure the response is a JSON.
![Screenshot from 2023-10-12 19-51-24](https://github.com/RileyManda/blog-app/assets/91142432/8d3e1a9c-14f9-4a4b-aed1-e7781a44a41f)